integration/spark/docker-compose.yml

```diff
@@ -1,17 +1,17 @@
 version: "3.7"
 services:
 # Jupyter Notebook with PySpark
-  notebook:
-    image: jupyter/pyspark-notebook:spark-3.1.2
-    ports:
-      - "8888:8888"
-    volumes:
-      - ./docker/notebooks:/home/jovyan/notebooks
-      - ./build:/home/jovyan/openlineage
-    links:
-      - "api:marquez"
-    depends_on:
-      - api
+  # notebook:
+  #   image: jupyter/pyspark-notebook:spark-3.1.2
+  #   ports:
+  #     - "8888:8888"
+  #   volumes:
+  #     - ./docker/notebooks:/home/jovyan/notebooks
+  #     - ./build:/home/jovyan/openlineage
+  #   links:
+  #     - "api:marquez"
+  #   depends_on:
+  #     - api
```

Comment:
Not sure why it's commented out in this PR.

Reply:
Please, just ignore it. It's not meant to be accepted yet. :-)